python/TestHarness/suppressions/errors.supp

```diff
@@ -189,3 +189,16 @@
    fun:*boost*math*
    ...
 }
+{
+   hypre_leaks
+   Memcheck:Leak
+   fun:hypre_HostMalloc
+   fun:hypre_MAlloc_core
+   fun:hypre_CAlloc
+   fun:hypre_IntArrayCreate
+   fun:hypre_BoomerAMGCoarseParmsHost
+   fun:hypre_BoomerAMGCoarseParms
+   fun:hypre_BoomerAMGSetup
+   fun:HYPRE_BoomerAMGSetup
+   fun:PCSetUp_HYPRE
+}
```

Comment:
```
test:aqueous_equilibrium.2species: ==1397864==    at 0x4C38CE4: calloc (vg_replace_malloc.c:762)
test:aqueous_equilibrium.2species: ==1397864==    by 0xBFF1366: hypre_HostMalloc (memory.c:174)
test:aqueous_equilibrium.2species: ==1397864==    by 0xBFF1366: hypre_MAlloc_core (memory.c:320)
test:aqueous_equilibrium.2species: ==1397864==    by 0xBFF1366: hypre_CAlloc (memory.c:806)
test:aqueous_equilibrium.2species: ==1397864==    by 0xBFF0F91: hypre_IntArrayCreate (int_array.c:26)
test:aqueous_equilibrium.2species: ==1397864==    by 0xBD8DCB6: hypre_BoomerAMGCoarseParmsHost (par_coarse_parms.c:89)
test:aqueous_equilibrium.2species: ==1397864==    by 0xBD8DD6C: hypre_BoomerAMGCoarseParms (par_coarse_parms.c:155)
test:aqueous_equilibrium.2species: ==1397864==    by 0xBD7D45F: hypre_BoomerAMGSetup (par_amg_setup.c:1455)
test:aqueous_equilibrium.2species: ==1397864==    by 0xBD5B04B: HYPRE_BoomerAMGSetup (HYPRE_parcsr_amg.c:47)
test:aqueous_equilibrium.2species: ==1397864==    by 0xAF9D118: PCSetUp_HYPRE (hypre.c:418)
test:aqueous_equilibrium.2species: ==1397864==    by 0xAFB7EBF: PCSetUp (precon.c:1017)
test:aqueous_equilibrium.2species: ==1397864==    by 0xADD778A: KSPSetUp (itfunc.c:408)
test:aqueous_equilibrium.2species: ==1397864==    by 0xADD97F8: KSPSolve_Private (itfunc.c:852)
test:aqueous_equilibrium.2species: ==1397864==    by 0xADDCA06: KSPSolve (itfunc.c:1086)
```

Reply:
@lindsayad  Do not understand why this suppress does not work for this case.

It is the same to me. My eyes do not work today?

Reply:
https://civet.inl.gov/job/986298/

Reply:
I will take a look and refresh my memory more on suppressions.

Reply:
I used valgrind's `--gen-suppressions`. I just looks like a more specific version of the one you already had so I dunno, but it definitely matches and resolves the MOOSE test failures. I'll check on chemical reactions now